src/screenshot.cpp

```diff
@@ -66,6 +67,9 @@ struct ScreenshotFormat {
 	ScreenshotHandlerProc *proc; ///< Function for writing the screenshot.
 };
 
+#define MKCOLOUR(x)         TO_LE32X(x)
+#define MKCOLOUR_XXXX(x)    (MKCOLOUR(0x01010101) * (uint)(x))
```

Comment:
Comment explaining what 0x01010101 is for would be great

Reply:
I actually have no idea why it should be multipled by 0x01010101. Perhaps it looks like a mask for making square image to diagonal for smallmap. If so, when I think about it, it does not seem necessary. (Minimap screenshot is taken well if ``MKCOLOUR_XXXX(x)`` is removed)
I'd like to listen your opinions :)